app/scripts/modules/google/loadBalancer/details/loadBalancerDetails.html

```diff
@@ -75,10 +75,10 @@ <h3 select-on-dbl-click>
         <div ng-if="!ctrl.isElSeven(loadBalancer) && loadBalancer.elb.dnsname">
           <dt>DNS Name</dt>
           <dd>
-            <a target=_blank href="//{{loadBalancer.elb.dnsname}}">{{loadBalancer.elb.dnsname}}</a>
+            <a target=_blank href="{{loadBalancer.elb.dns.protocol}}//{{loadBalancer.elb.dns.dnsname}}">{{loadBalancer.elb.dns.dnsname}}</a>
             <copy-to-clipboard
                 class="copy-to-clipboard copy-to-clipboard-sm"
-                text="{{loadBalancer.elb.dnsname}}"
+                text="{{loadBalancer.elb.dns.protocol}}//{{loadBalancer.elb.dns.dnsname}}"
                 tool-tip="Copy DNS Name to clipboard">
```

Comment:
Might want to change the tooltip to "Copy to clipboard", or omit the protocol on this link. We found users really just wanted to copy the DNS name itself without including a full link here.